project.js

```diff
@@ -386,30 +393,33 @@ function processProjectData(projectId, listOffset, callbackTrigger) {
         if (validProject && projXMLcode.length > 3) {
             // a footer to generate a watermark with the project's information at the bottom-right corner of the SVG
             // and hold project metadata.
-            var SVGfooter = '<text style="font-size:10px;fill:#555;" x="20" y="140">Parallax BlocklyProp Project</text>';
```

Comment:
Refactoring this into a separate function. I think it helps reduce the distractions to what the function is doing